### PR TITLE
[JENKINS-36720] - Fix SpotBugs warnings for "known as null" values

### PR DESCRIPTION
**Pull request by lacexd**
[JENKINS-36720] fixed a spotbug warning

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
Is it a Hacktoberfest pull request? https://jenkins.io/events/hacktoberfest

File: core/src/main/java/hudson/ClassicPluginStrategy.java [hunk 1]
```diff
@@ -273,7 +273,7 @@ private void fix(Attributes atts, List<PluginWrapper.Dependency> optionalDepende
      * @see DetachedPluginsUtil#getImpliedDependencies(String, String)
      */
     @Deprecated // since TODO
-    @Nonnull
+    @CheckForNull
```
Comment (oleg-nenashev) [inline core/src/main/java/hudson/ClassicPluginStrategy.java]: Hm? https://github.com/jenkinsci/jenkins/blob/0f73231f775e06105103fa9d0f23e20d7b122ac1/core/src/main/java/jenkins/plugins/DetachedPluginsUtil.java#L83 is marked as nonnull

**Comment by lacexd** [comment]
> Is it a Hacktoberfest pull request? https://jenkins.io/events/hacktoberfest

yes! :)

File: core/src/main/java/hudson/ClassicPluginStrategy.java (hunk 1 above)
Comment (lacexd) [reply core/src/main/java/hudson/ClassicPluginStrategy.java]: oh sorry, my bad.

Comment (oleg-nenashev) [reply core/src/main/java/hudson/ClassicPluginStrategy.java]: @lacedx So what are the next steps here? It looks like this change is not valid

**Comment by oleg-nenashev** [comment]
I suggest closing it if there is no follow-up

**Comment by varyvol** [comment]
Given there's been no answer to the proposed changes for some time now, we are closing this. Please feel free to reopen if you plan to continue working on this.

**Pull request closed without merging**